webapp/views/wpt-app.js

```diff
@@ -340,6 +375,32 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
     }
     return msg;
   }
+
+  computeBSFBannerMessage(isBSFCollapsed) {
+    if (isBSFCollapsed) {
+      return 'Browser Specific Failures graph (click the arrow to expand)';
+    }
+    return 'Browser Specific Failures graph (click the arrow to collapse)';
```

Comment:
```suggestion
    let state = isBSFCollapsed ? 'expand' : 'collapse';
    return `Browser Specific Failures graph (click the arrow to ${state})`;
```

('state' is a bad name, worth thinking of a better one)